kong/tools/ip.lua

```diff
@@ -0,0 +1,315 @@
+local ipmatcher  = require "resty.ipmatcher"
+local pl_stringx = require "pl.stringx"
+
+
+local gsub     = string.gsub
+local sub      = string.sub
+local fmt      = string.format
+local lower    = string.lower
+local find     = string.find
+local split    = pl_stringx.split
+local type     = type
+local ipairs   = ipairs
+local tonumber = tonumber
```

Comment:
These functions are build-in functions, should be in front of `gsub`

Reply:
done